app/assets/javascripts/responsive_table.js

```diff
@@ -0,0 +1,25 @@
+(function () {
```

Comment:
Did you know the app typically uses a custom Markdown rendering file for changes like this: https://github.com/buildkite/docs/blob/6859c165c5cf38cae6575bf8efad9ee8bead1ad3/app/models/page/renderer.rb

It looks like not all of them are Markdown tables, though, so maybe the JS is the answer. Happy to talk this through if you want to hash out the pros and cons.

Reply:
thanks @lachlanhardy no I wasn't aware of `renderer.rb`

This is handy! I would prefer to avoid JavaScript. I'll put this PR back into draft and have another go.